yoke/service.py

```diff
@@ -181,98 +118,15 @@ def flush(self):
         def close(self):
             self.device.close()
 
-
-zeroconf = Zeroconf()
-
-
-# Webserver to serve files to android client
-from http.server import HTTPServer, SimpleHTTPRequestHandler
 from threading import Thread
```

Comment:
Just a small nitpick: all imports could now be moved to the top of the file (unless there is reason to have it here).

Reply:
Not anymore. Splitting the code into several files really makes the dependencies much easier to understand than separating code sections with imports. 0fa305a3f1cca78206e50dc2466c923eb0e06401